Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.18
+
+RUN apk add --no-cache \
+	bash \
+	docker-cli \
+	tini
+
+COPY scripts/entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
+
+ENTRYPOINT ["/sbin/tini", "--", "/entrypoint.sh"]
+CMD [ "-h" ]
+
+COPY sbom-convert_*.apk /tmp/
+RUN apk add --no-cache --allow-untrusted /tmp/sbom-convert_*.apk
```

Comment:
i think goreleaser actually allows you to access the built binary artifact.
i can see how installing a APK can allow better version management on top of the API based image.

We can also go the route of a scrach based image with the single binary, just putting it out there see what people think..